.github/scripts/prepare_release.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Fail on first error.
+set -e
+
+releaseLevel="$1"
+
+oldVersion="$(node -pe 'require("./package.json").version')"
+npx standard-version --release-as "$releaseLevel"
+
+cd selenium
+npx standard-version --release-as "$releaseLevel"
+cd ..
```

Comment:
You no longer need to do this (since you tell `standard-version` to bump both versions).